ios/Runner/Info.plist

```diff
@@ -43,5 +43,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>We will use the microphone to improve your app experience</string>
```

Comment:
i add this microphone use, but in the future if it should not be necessary, we could remove it.